tensorboard/webapp/widgets/data_table/header_cell_component_test.ts

```diff
@@ -76,26 +80,29 @@ describe('header cell', () => {
       name: 'run',
       order: SortingOrder.ASCENDING,
     };
+    fixture.componentInstance.hparamsEnabled =
+      input.hparamsEnabled === undefined ? true : input.hparamsEnabled;
+    fixture.componentInstance.controlsEnabled =
+      input.controlsEnabled === undefined ? true : input.controlsEnabled;
```

Comment:
nit: This seems like a good use case for the null coalesce operator
```suggestion
    fixture.componentInstance.hparamsEnabled = input.hparamsEnabled ?? true;
    fixture.componentInstance.controlsEnabled = input.controlsEnabled ?? true;
```